lti_consumer/lti_1p1/contrib/django.py

```diff
@@ -123,6 +125,14 @@ def lti_embed(
     }
     context.update({'lti_parameters': lti_parameters})
 
+    # emit tracking event
+    event = {
+        'lti_version': lti_parameters.get('lti_version'),
+        'user_roles': lti_parameters.get('roles'),
+        'launch_url': lti_consumer.lti_launch_url,
+    }
+    track_event('embed.launch_request', event)
```

Comment:
right now you could lti.track_even with lots of different data - should track_event take a set of defined fields instead of just a freeform event?

Reply:
I'm thinking freeform since some 'defined fields' such as course id / context get tacked on deeper in the library. Each event itself may be totally different in what we want to store.